src/resize-handle.directive.ts

```diff
@@ -43,9 +53,9 @@ export class ResizeHandleDirective implements OnInit, OnDestroy {
 
   constructor(
     private renderer: Renderer2,
-    private element: ElementRef,
+    public element: ElementRef,
```

Comment:
Does this need to be public?

Reply:
just made it public for unit tests sake
no real need, just saw same pattern in ResizableDirective and decided to shortcut same way

Reply:
OK can we keep it private and get the element a different way in the test (via `querySelector` or `debugElement`). Users have an annoying way of using all the apis you didn't expect to expose 😅

Reply:
totally agree, updated